ui/apps/platform/src/Containers/Clusters/ClusterDeployment.tsx

```diff
@@ -55,10 +55,14 @@ function ClusterDeployment({
                         spaceItems={{ default: 'spaceItemsMd' }}
                     >
                         <Title headingLevel="h3">1. Download files</Title>
-                        <Text>Download the required configuration files, keys, and scripts.</Text>
+                        <Text>
+                            Download the required configuration files, keys, and scripts. Enabling
+                            automatic upgrades creates a powerful service account in your secured
+                            cluster that will be used to perform the upgrades.
+                        </Text>
                         <Switch
-                            label="Cluster is configured to allow future automatic upgrades"
-                            labelOff="Click to configure cluster to allow future automatic upgrades"
+                            label="Automatic upgrades ON: Secured Clusters are upgraded automatically to match Central's version."
+                            labelOff="Automatic upgrades OFF: Secured clusters are not upgraded automatically."
```

Comment:
I am obviously late to the party, but I think, we may need to give this text another look.
The issues are:
1. Automatic upgrades are enabled on the cluster page, not here where we download the bundle,
2. This on/off applies only for the legacy installation method, not for other methods. Maybe clarifying this here would be a good addition?
3. What we are turning on/off here is the _possibility_ of doing an upgrade
    1. If this is on, the upgrade will most probably succeed when someone triggers them or they are being triggered automatically
    2. If this is off, then the upgrades can still be triggered manually, but they will fail. I am not sure if they still could be triggered automatically, but if they do, they will fail (we would need to check that).